services/feed-manager-service/feed-manager-core/src/main/java/com/thinkbiganalytics/feedmgr/service/category/SimpleCategoryCache.java

```diff
@@ -19,6 +19,26 @@
  * #L%
  */
 
+/*-
+ * #%L
+ * thinkbig-feed-manager-core
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)